frontend/package.json

```
@@ -14,7 +14,10 @@
     "test": "jest",
     "lint": "yarn eslint . --ext .js,.ts,.tsx",
     "clean": "rm -rf .next",
-    "generate": "graphql-codegen --config codegen.yaml"
+    "generate": "graphql-codegen --config codegen.yaml",
+    "cypress:local": "concurrently \"yarn start:stubs\" \"NODE_ENV=development node server.js\" \"cypress open --config-file ./cypress.json\"",
+    "start:stubs": "node stubServer",
+    "cypress:ci": "yarn start:stubs & yarn dev"
```

```suggestion
    "start:stubs": "node stubServer"
```
and just use yarn dev in ci